provisioning/miniprov/hare_mp/main.py

```diff
@@ -510,7 +516,9 @@ def config(args):
     try:
         url = args.config[0]
         filename = args.file[0] or '/var/lib/hare/cluster.yaml'
-        save(filename, generate_cdf(url))
+        motr_md_path = '/opt/seagate/cortx/motr/conf/motr_hare_keys.json'
```

Comment:
May be we can have a configuration file for such kind of constants, so that the values can be easily configurable and we don't have to change the code. We need a bit of design for that and its out of scope of this PR.